dgbpy/hdf5.py

```diff
@@ -661,6 +657,17 @@ def getInfo( filenm, quick ):
     retinfo.update({plfdictstr: odhdf5.getText(info,'Model.Type')})
   if  odhdf5.hasAttr(info,versionstr):
     retinfo.update({versiondictstr: odhdf5.getText(info,versionstr)})
+  if img2img & isclassification:
+    groups = retinfo[exampledictstr].keys()
```

Comment:
First try to read the value from the ++info++ section, like before:
```
hasunlabels = False
if odhdf5.hasAttr(info, withunlabeleddictstr):
  hasunlabels = odhdf5.getBoolValue( info, withunlabeleddictstr )
```
Then only do your implementation below as part of the else block if that same if statement